.github/rsi-schema.json

```diff
@@ -1,185 +1,244 @@
 {
-   "$schema":"http://json-schema.org/draft-07/schema",
-   "default":{
-      
-   },
-   "description":"JSON Schema for SS14 RSI validation.",
-   "examples":[
-      {
-         "version":1,
-         "license":"CC-BY-SA-3.0",
-         "copyright":"Taken from CODEBASE at COMMIT LINK",
-         "size":{
-            "x":32,
-            "y":32
-         },
-         "states":[
-            {
-               "name":"basic"
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "default": {
+    },
+    "description": "JSON Schema for SS14 RSI validation.",
+    "examples": [
+        {
+            "version": 1,
+            "license": "CC-BY-SA-3.0",
+            "copyright": "Taken from CODEBASE at COMMIT LINK",
+            "size": {
+                "x": 32,
+                "y": 32
             },
-            {
-               "name":"basic-directions",
-               "directions":4
-            },
-            {
-               "name":"basic-delays",
-               "delays":[
-                  [
-                     0.1,
-                     0.1
-                  ]
-               ]
+            "states": [
+                {
+                    "name": "basic",
+                    "license": "CC-BY-SA-3.0",
+                    "copyright": "Taken from CODEBASE at COMMIT LINK"
```

Comment:
Would be nice to also remove "put links into copyright field" from here given we do have that now